Remotion/Web/Development.WebTesting/WebDriver/Configuration/Firefox/FirefoxBinariesProvider.cs

```diff
@@ -97,6 +99,7 @@ private string GetDriverPathAndDownloadIfMissing (string firefoxPath)
 
       if (!DriverExists (driverVersion))
         DownloadDriver (driverDownloadUrl, versionedGeckoDriverDirectory);
+      ServicePointManager.SecurityProtocol = initialSecurityProtocol;
```

Comment:
Shouldn't this be in a `finally` block?

Reply:
You're right, I adjusted it accordingly.